web_widget_bokeh_chart/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 ForgeFlow S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Web Widget Bokeh Chart",
+    "category": "Hidden",
+    "summary": "This widget allows to display charts using Bokeh library.",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA), " "Creu Blanca",
+    "version": "18.0.1.0.0",
+    "maintainers": ["LoisRForgeFlow", "ChrisOForgeFlow"],
```

Comment:
Could you take over as co-maintainer instead of ChrisOForgeFlow?

Reply:
Done, thanks